runtime/oti/vm_api.h

```diff
@@ -1489,9 +1489,9 @@ areValueTypesEnabled(J9JavaVM *vm);
 
 
 /**
- * @brief Queries if -XX:ValueBasedClassCheck=exception or -XX:ValueBasedClassCheck=exception are found in the CML
+ * @brief Queries if -XX:DiagnoseSyncOnValuedBasedClasses=1 or -XX:DiagnoseSyncOnValuedBasedClasses=2 are found in the CML
```

Comment:
This comment could also refer to the runtime flags since the code actually checks those.

Reply:
Comment added.